### ISSUE TITLE
IBM comment 3 on 2.2.6 Accessible Authentication

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag21#442]
The SC invalidates various two-factor authentication processes that rely on the user to transcribe a numeric or alpha-numeric string. IBM would like research-based clarity on whether any amount of transcription is considered an impediment to users with cognitive disabilities such that it warrants exclusion of transcription as an accepted technique. If existing research shows that there is a certain number of transcribed characters that could be considered achievable, that is what the guidance should be based on. In example, supporting evidence from one study cited in the [Issue thread for 2.2.6](https://github.com/w3c/wcag21/issues/23#issuecomment-333952517) (Computerized Measurement of Implicit Short Term Memory Retention in a Digit Copying Task) showed that all test subjects were able to complete 5-digit transcription -- and five digits was not established as a ceiling.

**Comment by awkawk** [comment]
Official WG response: Thank you for your comment. This SC has been removed from the draft and will be able to be considered in a future revision.
